tests/foreman/ui/test_rhc.py

```diff
@@ -41,9 +41,7 @@ def fixture_enable_rhc_repos(module_target_sat):
 def module_rhc_org(module_target_sat):
     """Module level fixture for creating organization"""
     if settings.rh_cloud.crc_env == 'prod':
-        org = module_target_sat.api.Organization(
-            name=settings.rh_cloud.organization or gen_string('alpha')
-        ).create()
+        org = module_target_sat.api.Organization(name=settings.rh_cloud.organization).create()
```

Comment:
Having a same name of the manifest will create a conflict while creating another manifest.

Hence please combine the name with the random string like:
name=f"{settings.rh_cloud.organization}_{gen_string('alpha')}"

Reply:
I was thinking the same but then realise doing that would break the name we set with automation `satellite-jenkins/-/blob/master/src/resources/components/redhatCloudPipeline.groovy?ref_type=heads#L36` that's why I'm not adding random string there. But I realize it'll cause issue with the test when it runs in normal automation pipeline. I added other fix to avoid it, please re-review:)

Reply:
we probably don't need to do any changes in jenkins